src/control/run_go_tests.sh

```diff
@@ -1,6 +1,61 @@
 #!/bin/bash
 ## Run linters across control plane code and execute Go tests
-set -e
+set -x
+
+function find_build_source()
+{
+	BASE=$PWD
+	
+	while true
+	do
+		path=`realpath ${BASE}`
```

Comment:
(lint) Use $(..) instead of deprecated `..` [SC2006]
(lint) Double quote to prevent globbing and word splitting. [SC2086]